extensions/historical-imagery/tests/historical-imagery_collection.test.js

```diff
@@ -227,4 +244,21 @@ o.spec('Historical Imagery Extension Collection', () => {
     // then
     o(valid).equals(true);
   });
+
+  o("Summaries with no 'summaries' should fail validation", async () => {
```

Comment:
"Summaries" should probably be "Example" here.

Reply:
Oops, thanks!